mypy/test/data.py

```diff
@@ -610,12 +622,15 @@ def repr_failure(self, excinfo: Any) -> str:
 
 
 class DataSuite:
+    files = None  # type: typing.ClassVar[List[str]]
```

Comment:
I'd rather not have the `ClassVar` annotations here and elsewhere -- they add noise and distract from the main intent of the code. It would be okay to just add a comment here mentioning that all of these should be treated as class variables.